core/src/main/java/nl/nn/adapterframework/stream/SerializableFileReference.java

```diff
@@ -103,7 +103,7 @@ public static SerializableFileReference of(byte[] data) throws IOException {
 	 * @throws IOException If the {@link Reader} cannot be read or a temporary file cannot be created / written to.
 	 */
 	public static SerializableFileReference of(Reader in, String charset) throws IOException {
-		try (InputStream is = new ReaderInputStream(in, charset)) {
+		try (InputStream is = ReaderInputStream.builder().setReader(in).setCharset(charset).get()) {
```

Comment:
Pfff het is er niet beter van geworden, ik vraag mij af wie besloten had dat dit de nieuwe 'way to go' is....

Reply:
Tsja. Ik zag dat de constructors deprecated waren dus ik dacht, ik pas het toch maar aan.